### PR TITLE
refactor: move different cognitive services into sub packages

### PR DESCRIPTION
**Pull request by serena-ruan**
## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/documentation/estimators/cognitive/_MAD.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Festimators%2Fcognitive%2F_MAD.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AnomalyDetection.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AnomalyDetection.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AzureSearch.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AzureSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_BingImageSearch.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_BingImageSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_ComputerVision.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_ComputerVision.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Face.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Face.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_FormRecognizer.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_FormRecognizer.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_SpeechToText.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_SpeechToText.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_TextAnalytics.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_TextAnalytics.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Translator.md](https://github.com/microsoft/SynapseML/blob/6674cdf76eb839080418ef9f43735a54213bef2e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Translator.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/documentation/estimators/cognitive/_MAD.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Festimators%2Fcognitive%2F_MAD.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AnomalyDetection.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AnomalyDetection.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AzureSearch.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AzureSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_BingImageSearch.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_BingImageSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_ComputerVision.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_ComputerVision.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Face.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Face.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_FormRecognizer.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_FormRecognizer.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_SpeechToText.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_SpeechToText.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_TextAnalytics.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_TextAnalytics.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Translator.md](https://github.com/microsoft/SynapseML/blob/9612f6d9d92e323fcfb23a9d83aaf96109cfc50d/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Translator.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/AnomalyDetection.scala [hunk 1]
```diff
@@ -1,9 +1,10 @@
 // Copyright (C) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in project root for information.
 
-package com.microsoft.azure.synapse.ml.cognitive
+package com.microsoft.azure.synapse.ml.cognitive.anomaly
```
Comment (svotaw) [inline cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/AnomalyDetection.scala:4]: aren't these namespace changes breaking changes?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/documentation/estimators/cognitive/_MAD.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Festimators%2Fcognitive%2F_MAD.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AnomalyDetection.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AnomalyDetection.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AzureSearch.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AzureSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_BingImageSearch.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_BingImageSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_ComputerVision.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_ComputerVision.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Face.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Face.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_FormRecognizer.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_FormRecognizer.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_SpeechToText.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_SpeechToText.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_TextAnalytics.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_TextAnalytics.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Translator.md](https://github.com/microsoft/SynapseML/blob/d66a5e8448282a63d240ee453f10c78bf006d49f/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Translator.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/AnomalyDetection.scala (hunk 1 above)
Comment (serena-ruan) [reply cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/AnomalyDetection.scala:4]: For scala yes, we need to change namespaces when importing them. But for python, I'll make it backward compatible.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/documentation/estimators/cognitive/_MAD.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Festimators%2Fcognitive%2F_MAD.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AnomalyDetection.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AnomalyDetection.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AzureSearch.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AzureSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_BingImageSearch.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_BingImageSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_ComputerVision.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_ComputerVision.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Face.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Face.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_FormRecognizer.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_FormRecognizer.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_SpeechToText.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_SpeechToText.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_TextAnalytics.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_TextAnalytics.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Translator.md](https://github.com/microsoft/SynapseML/blob/2c941afe84ab2e8ed5aa1feeeb2924d73851e23e/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Translator.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
Lovely work! I dont see the python backcompat stuff in the PR yet but everything else looks great. Especially liked how you remembered to update the test matrix in pipeline.yml

**Comment by serena-ruan** [comment]
> Lovely work! I dont see the python backcompat stuff in the PR yet but everything else looks great. Especially liked how you remembered to update the test matrix in pipeline.yml

In PyCodegen file, I made a change to __init__.py file so that it also imports subpackages in cognitive namespace :D

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/documentation/estimators/cognitive/_MAD.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Festimators%2Fcognitive%2F_MAD.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AnomalyDetection.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AnomalyDetection.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AzureSearch.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AzureSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_BingImageSearch.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_BingImageSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_ComputerVision.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_ComputerVision.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Face.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Face.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_FormRecognizer.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_FormRecognizer.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_SpeechToText.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_SpeechToText.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_TextAnalytics.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_TextAnalytics.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Translator.md](https://github.com/microsoft/SynapseML/blob/1df69ef372045f59789c908220a7a28ebc2fec00/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Translator.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/documentation/estimators/cognitive/_MAD.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Festimators%2Fcognitive%2F_MAD.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AnomalyDetection.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AnomalyDetection.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_AzureSearch.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_AzureSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_BingImageSearch.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_BingImageSearch.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_ComputerVision.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_ComputerVision.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Face.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Face.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_FormRecognizer.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_FormRecognizer.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_SpeechToText.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_SpeechToText.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_TextAnalytics.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_TextAnalytics.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |
| [website/docs/documentation/transformers/cognitive/_Translator.md](https://github.com/microsoft/SynapseML/blob/d8721d537e2b069e8b596446fd7f54904aac7f2b/website%2Fdocs%2Fdocumentation%2Ftransformers%2Fcognitive%2F_Translator.md) |   |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**